src/commcare_cloud/ansible/roles/common_installs/tasks/main.yml

```diff
@@ -92,3 +93,4 @@
     name: virtualenv-clone
     version: 0.5.1
     executable: pip2
```

Comment:
@dannyroberts not sure if you're tracking our `pip2` instances of where we will need to change to upgrade to python3

Reply:
Oh interesting. These are basically the system dependencies we use rather than the application dependencies, and I see three places where we do that. Seems like they're fairly decoupled both from each other and from the commcare python 3 upgrade, but I agree we should try and remove the python 2 dependency here as well. Making myself a ticket for that: https://dimagi-dev.atlassian.net/browse/PLAT-200